tests/dragonfly/pymemcached_test.py

```diff
@@ -67,7 +68,9 @@ def test_noreply_pipeline(df_server: DflyInstance, memcached_client: MCClient):
     assert memcached_client.get_many(keys) == {k: v.encode() for k, v in zip(keys, values)}
 
     info = Redis(port=df_server.port).info()
-    assert info["total_pipelined_commands"] > 0  # sometimes CI is slow
+    if info["total_pipelined_commands"] == 0:
```

Comment:
Can you also bump up the range slightly ? It's 2k on line 59, can you plz try 4 or 8 k?